xcode/X16/iOS/GameControlViewController.xib

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
XIBs are impossible to review. :( I'm trusting you this is complex enough that you don't want to build any of this in code... :)

Reply:
Hoping that these are going to be temporary.